merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/GetPlanDatasetsAction.java

```diff
@@ -35,7 +37,11 @@ public List<PlanDatasetRecord> get(final PlanId planId) throws SQLException {
     while (resultSet.next()) {
       final var datasetId = resultSet.getLong(1);
       final var offsetFromPlanStart = parseOffset(resultSet, 2);
-      records.add(new PlanDatasetRecord(planId.id(), datasetId, offsetFromPlanStart));
+      final var associatedSimulationDatasetId = Optional
+          .ofNullable(resultSet.getString(3))
+          .map(Long::valueOf)
+          .map(SimulationDatasetId::new);
```

Comment:
Minor, but a suggestion based on how we get an `Optional<Long>` in [`GetSpanRecords`](https://github.com/NASA-AMMOS/aerie/blob/develop/merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/GetSpanRecords.java#L49):
```suggestion
      final Optional<SimulationDatasetId> associatedSimulationDatasetId = 
          resultSet.getObject(3) == null ? 
              Optional.empty() : 
              Optional.of(new SimulationDatasetId(resultSet.getLong(3)));
```